src/utils/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const ACTION_TYPE: { [key: string]: string } = {
+  newUserCreated: 'New user created',
+  grUpdate: 'GR updated',
+  grAndPatientRecordLinked: 'GR and Patient record linked',
+  grAndPatientRecordLinkApproved: 'GR and Patient record link approved'
+}
```

Comment:
```suggestion
export const ACTION_TYPE: { [key: string]: string } = {
  newUserCreated: 'New user created',
  grUpdate: 'GR updated',
  grAndPatientRecordLinked: 'GR and Patient record linked',
  grAndPatientRecordLinkApproved: 'GR and Patient record link approved'
}
```
Can we add a "enum" type to enumerate the different action types : 'newUserCreated', 'grUpdate', ...